modules/services/display_svc/src/display_svc.cpp

```diff
@@ -0,0 +1,41 @@
+
+
+#include <iostream>
+
+#include "display_svc.h"
+
+namespace ssw
+{
+DisplaySvc::DisplaySvc(kpsr::Environment *environment)
+    : Service(environment, "stream_switcher_service") {}
+
+void DisplaySvc::start()
+{
+  spdlog::info("Start display service");
+}
+
+void DisplaySvc::stop()
+{
+  spdlog::info("Stop display service");
+}
+
+void DisplaySvc::execute()
+{
+  //spdlog::info("Execute");
+   //read img to _currentImage
+  //showImage(_currentImage);
+  
+}
+
+void DisplaySvc::showImage(kpsr::vision_ocv::ImageData &image)
+{
+  spdlog::info("Showing Image {}", image.seq);
+  if (image.img.empty())
+  {
+    spdlog::info("Could not read the image");
+    return;
+  }
+  cv::imshow("Show image", image.img);
```

Comment:
Why do we have the `imshow` are and also in the read service?
Same problem with the SimpleReadService: use different image names for different instances.

Reply:
Not defined very well in this branch.